deploy/infrastructure/utils/variables.py

```diff
@@ -18,38 +18,66 @@
 """
 
 # Variables per project
-# terraform-google-kubernetes
+# dependencies/terraform-commons-dss
+COMMONS_DSS_VARIABLES = [
+    "image",
+    "authorization",
+    "enable_scd",
+    "should_init",
+    "desired_rid_db_version",
+    "desired_scd_db_version",
+    "crdb_locality",
+    "crdb_external_nodes",
+    "kubernetes_namespace",
+    "app_hostname",
+    "crdb_hostname_suffix",
+]
+
+# dependencies/terraform-google-kubernetes
 GOOGLE_KUBERNETES_VARIABLES = [
     "google_project_name",
     "google_zone",
     "google_dns_managed_zone_name",
-    "google_kubernetes_storage_class",
     "app_hostname",
     "crdb_hostname_suffix",
     "cluster_name",
     "node_count",
```

Comment:
Perhaps for future movement to `COMMON_KUBERNETES_VARIABLES`?  (I'm thinking it would be nice to have as few cloud-provider-specific things as practical)

Reply:
Makes sense, thank you.